docs/eventing/getting-started.md

```diff
@@ -62,7 +61,7 @@ demonstrate how you can configure your event producers to target a specific cons
 1. To deploy the `hello-display` consumer to your cluster, run the following
    command:
 
-     ```
+     ```yaml
      kubectl -n event-example apply -f - << EOF
      apiVersion: apps/v1
      kind: Deployment
```

Comment:
is there a reason we use the plain k8s APIs, rather than, say `ksvc`s?

Reply:
I think the idea is so that you don't need Serving to use the Eventing examples